### PR TITLE
WRR-11077: Fixed Scroller to prevent the native scrolling behavior caused by keydown events when the popup is opened

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
When the Scroller has a button that opens FixedPopupPanel. The issue is that when press the down key at the moment the popup is opened the scroller behind the FixedPopupPanel scrolls down.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
To fix this issue, we have to prevent the native scrolling behavior caused by keydown events.
I created preventScroll function which call `preventDefault` when a keydown event is caused by a directional key and the spotlight is paused.
I added this preventScroll function to useThemeScroll so that it can be passed to ui/Scroller and used as prop.

Also I added the issue case in Scroller qa-sampler

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-11077

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please fix the travis error and update PR title and description accordingly.

File: samples/sampler/stories/qa/Scroller.js [hunk 1]
```diff
@@ -1284,3 +1284,35 @@ export const WithContentContainerOnFocus = () => {
 };
 
 WithContentContainerOnFocus.storyName = 'With Content Container';
+
+export const WithFixedPopupPanels = () => {
+	const [open, setOpenState] = useState(false);
+
+	const handleOpen = useCallback(() => {
+		setOpenState(true);
+	}, [setOpenState]);
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Scroller.js:1293]: We don't have to add setter as dependency here.
Please refer to https://react.dev/reference/react/useState#setstate-caveats.

File: samples/sampler/stories/qa/Scroller.js [hunk 2]
```diff
@@ -1284,3 +1284,35 @@ export const WithContentContainerOnFocus = () => {
...
+
+	const handleClose = useCallback(() => {
+		setOpenState(false);
+	}, [setOpenState]);
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Scroller.js:1297]: here too.

File: samples/sampler/stories/qa/Scroller.js [hunk 3]
```diff
@@ -1284,3 +1284,35 @@ export const WithContentContainerOnFocus = () => {
...
+	return (
+		<>
+			<Scroller>
+				<Button onClick={handleOpen}>
+					Open FixedPopupPanels
+				</Button>
+				<div style={{height: ri.scaleToRem(2400)}} />
+			</Scroller>
+			<FixedPopupPanels open={open} onClose={handleClose}>
+				<FixedPopupPanels.Panel>
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Scroller.js:1308]:
```suggestion
				<Panel>
```

File: samples/sampler/stories/qa/Scroller.js [hunk 4]
```diff
@@ -1284,3 +1284,35 @@ export const WithContentContainerOnFocus = () => {
...
+					<Header title="Panel" />
+					<Item>Item A</Item>
+					<Item>Item B</Item>
+				</FixedPopupPanels.Panel>
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Scroller.js:1312]:
```suggestion
				</Panel>
```

File: .travis.yml [hunk 5]
```diff
@@ -16,7 +16,7 @@ install:
     - npm install
     - npm link
     - popd
-    - git clone --branch=release/4.9.x.develop --depth 1 https://github.com/enactjs/enact ../enact
+    - git clone --branch=feature/WRR-11077-4.9 --depth 1 https://github.com/enactjs/enact ../enact
```
Comment (MikyungKim) [inline .travis.yml:19]: Don't forget to revert the change before merging.

File: samples/sampler/stories/qa/Scroller.js (hunk 4 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/Scroller.js:1312]: It's fixed.

File: .travis.yml (hunk 5 above)
Comment (juwonjeong) [reply .travis.yml:19]: It's fixed.

File: samples/sampler/stories/qa/Scroller.js (hunk 3 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/Scroller.js:1308]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM
